### PR TITLE
Update doc to reflect User properties (instead of methods)

### PR DESCRIPTION
**Pull request by ChaoticMind**
Changed some leftover instances of "methods" to say "properties" instead.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -66,32 +66,31 @@ will continue.)
...
-    Returns `True` if this is an anonymous user. (Actual users should return
+    Is `True` if this is an anonymous user. (Actual users should return
     `False` instead.)
 
 `get_id()`
-    Returns a `unicode` that uniquely identifies this user, and can be used
+    Is a `unicode` that uniquely identifies this user, and can be used
     to load the user from the `~LoginManager.user_loader` callback. Note
     that this **must** be a `unicode` - if the ID is natively an `int` or some
     other type, you will need to convert it to `unicode`.
```
Comment (davidism) [inline docs/index.rst]: This is still a method.

**Comment by davidism** [comment]
`get_id()` is still a method, so the subsequent change to refer to everything as "properties" is not valid. Perhaps "attributes" is better, but I don't really see this as confusing anyway. Properties are still methods, and still return values.

**Comment by ChaoticMind** [comment]
Agreed, updated. Thanks.

**Comment by julian-r** [comment]
That is from a python perspective cool. BUT you are breaking the API in a minor version, that is really stupid!

**Comment by maxcountryman** [comment]
@julian-r as has already been said many times: flask-login is pre-1.0: there is no rule which states minor versions won't make breaking changes. These changes are well documented and it's up to library consumers to use this documentation to stay informed.

Your tone is rude and inappropriate for this forum: it disrespects the time and effort myself and others have put into this project. Please don't come here with that kind of comment in the future.

**Pull request merged**